Sources/AlgoliaSearch-Client/Query.swift

```diff
@@ -1106,4 +1128,10 @@ open class Query : AbstractQuery {
         get { return AbstractQuery.toNumber(self.percentileComputation) }
         set { self.percentileComputation = newValue?.boolValue }
     }
+    
+    @objc(sortFacetValuesBy)
+    public var z_objc_sortFacetValuesBy: String? {
+        get { return sortFacetValuesBy?.rawValue }
+        set { sortFacetValuesBy = newValue == nil ? nil : SortFacetValuesBy(rawValue: newValue!) }
```

Comment:
I would make it explicit with an if or guard. Even though is "longer" it's safer. Here, a miss matched equal could "break" the code

Reply:
discussed 1:1